pkgs/by-name/ty/typos/package.nix

```diff
@@ -17,6 +20,19 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-WuYAE2msctMf+7C1h5pNGyjrEInnT6PxfZ94bZhzB5w=";
 
+  passthru = {
+    tests.version = testers.testVersion {
+      inherit version;
+
+      package = typos;
+      command = ''
+        typos --version
+      '';
```

Comment:
```suggestion
      command = "typos --version";
```

Reply:
Sorry, I was guessing this style is okay because of nixfmt-rfc-style did not alert. 🙇‍♂️  Fixed in https://github.com/NixOS/nixpkgs/pull/354982/commits/579e0cd3d032073777113a35a308f06edeb4a896 (Or should I squash in above commits?)

Reply:
Yes squash it :)

Reply:
Thank you! Updated in https://github.com/NixOS/nixpkgs/pull/354982/commits/7e70402d2d0a23a8ce1062111cd734410900fb93